autovalet_system/scripts/slam.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+sim=$1
+if [ "$sim" = "true" ] 
+then
+    roslaunch autovalet_gazebo setup_slam_test.launch world_file:=state_machine.world x:=0 y:=6 1>> $HOME/slam_log.txt 2>&1
+else
+    roslaunch autovalet_husky integrated_sensor.launch 1>> $HOME/sensor_log.txt 2>&1
+    roslaunch autovalet_husky valet_rtab.launch 1>> $HOME/slam_log.txt 2>&1
```

Comment:
are these still two lines because we couldn't test it on hardware yet?

Reply:
no, this should work. i just made it so we can have a separate log file for `integrated_sensor` and `valet_rtab`

Reply:
why do we need two files for hardware but one for simulation only?